statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -374,9 +574,10 @@ def predict(self, params, start=None, end=None, lags=1, trend='c'):
             return predictedvalues
 
         # fit out of sample
-        y = y[-k_ar:]
+        Y = Y[-k_ar:]
         coefs = params[k_trend:].reshape((k_ar, k, k)).swapaxes(1,2)
-        predictedvalues[pv_end:] = forecast(y, coefs, intercept, out_of_sample)
+        predictedvalues[pv_end:] = forecast(Y, coefs, trend_coefs,
+                                            out_of_sample)
         return predictedvalues
 
     def fit(self, maxlags=None, method='ols', ic=None, trend='c',
```

Comment:
If we need to, add a new fit method that does the specification search, standalone function is above in module.

Aside: information criteria would also work for trend order selection